content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -183,6 +183,22 @@ into the HTTP connection manager filter chain. The local rate limit filter's [to
 is configured to allow 10 requests/min. The filter is also configured to add an `x-local-rate-limit`
 response header to requests that are blocked.
 
+{{< tip >}}
+The statistics mentioned [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enabled them with the following annotations during deployment:
```

Comment:
I'm confused by the word `here`, especially since you don't really mean `on this page`. This should be updated to something that describes the link. Something like mentioned on the [Envoy rating limiting page] might be better.

Reply:
updated the content.